proto/stride/records/records.proto

```diff
@@ -72,6 +76,19 @@ message HostZoneUnbonding {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable) = false
   ];
+  string st_tokens_to_burn = 8 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string native_tokens_to_unbond = 9 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string claimable_native_tokens = 10 [
```

Comment:
Do you think we need to store this, vs just using the last unbonding time in `unbonding_time` (i.e. assume all native tokens aren't claimable until the last unbonding finishes)? 

Also possible this is described in more detail in another PR, in which case please ignore!

Reply:
Good question! We definitely don't need it, but I figured that now that we have different rules around decrementing these amounts, it's better to just be explicit (at the cost of adding an additional field).

Feels much more likely to footgun ourselves if we start having different implicit rules around when a field should be set or when it's used as a counter